packages/web-runtime/src/components/UploadInfo.vue

```diff
@@ -167,13 +170,24 @@ export default defineComponent({
     runningUploads: 0, // all uploads (not files!) that are in progress currently
     bytesTotal: 0,
     bytesUploaded: 0,
+    uploadSpeed: 0,
     filesInEstimation: {},
     timeStarted: null,
     remainingTime: undefined
   }),
   computed: {
     ...mapGetters(['configuration']),
 
+    uploadDetails() {
+      if (this.uploadSpeed === 0) {
+        return ''
+      }
+      const uploadedBytes = filesize(this.bytesUploaded)
+      const totalBytes = filesize(this.bytesTotal)
+      const currentUploadSpeed = filesize(this.uploadSpeed)
+      // needs $gettext interpolation
+      return `${uploadedBytes} of ${totalBytes} (${currentUploadSpeed}/s)`
```

Comment:
that's a `TODO` that should be solved before merging. :-) fyi, in the vue3 version of gettext the interpolation happens directly in `$gettext` (grep for examples in the code)

Reply:
Updated, thx